Source/Backlang.Codeanalysis/Parsing/AST/Symbols.cs

```diff
@@ -13,4 +13,8 @@ public static class Symbols
     public static readonly Symbol Range = GSymbol.Get("'..");
     public static readonly Symbol ToExpand = GSymbol.Get("'to_expand'");
     public static readonly Symbol TypeLiteral = GSymbol.Get("#type");
+
+    public static readonly Symbol Float16 = GSymbol.Get("#float16");
+    public static readonly Symbol Float32 = GSymbol.Get("#float32");
```

Comment:
why not use CodeSymbols.Float and CodeSymbols.Double?

Reply:
There is only `CodeSymbols.Double`. Float and Half doesnt exists and for being in one style with the int and uints and the style of the language itself i used the `float16` to `float64`